### PR TITLE
Allow to configure the location fo the local maven repo

### LINKED ISSUE #358: Allow to configure the location fo the local maven repo

### PR DESCRIPTION
**Pull request by lburgazzoli**
Fixes #358

### CODE REVIEW

File: pkg/cmd/install.go [hunk 1]
```diff
@@ -140,6 +142,9 @@ func (o *installCmdOptions) install(cmd *cobra.Command, args []string) error {
 				}
 			}
 		}
+		if o.localRepository != "" {
```
Comment (nicolaferraro) [inline pkg/cmd/install.go]: Unneeded check

File: deploy/platform-cr.yaml [hunk 2]
```diff
@@ -8,3 +8,4 @@ spec:
   build:
     camelVersion: "2.23.1"
     baseImage: "fabric8/s2i-java:3.0-java8"
+    localRepository: "/tmp/artifacts/m2"
```
Comment (nicolaferraro) [inline deploy/platform-cr.yaml]: Better leaving it blank here and providing a default when initializing the platform. There are platforms that will not be created by `kamel install` and don't contain this option. Same is for the `baseImage` and also `camelVersion`, do they have a sensible default?

When initializing the platform, we may check for empty values and pick them from this resource file. Wdyt?

Comment (lburgazzoli) [reply deploy/platform-cr.yaml]: I do agree, what about opening a new issue for the defaults ?

Comment (nicolaferraro) [reply deploy/platform-cr.yaml]: Ok, it seems common to other props

**Pull request merged**